Descripion: Fix an incompatibility with quickcheck 2.6
Authot: Sven Bartscher <sven.bartscher@weltraumschlangen.de>
Forwarded: yes
Last-Update: 2014-08-25
--- a/tests/haskell/Network/WebSockets/Server/Tests.hs
+++ b/tests/haskell/Network/WebSockets/Server/Tests.hs
@@ -25,6 +25,7 @@
 import           Test.HUnit                     (Assertion, assert, (@=?))
 import           Test.QuickCheck                (Arbitrary, arbitrary)
 import           Test.QuickCheck.Gen            (Gen (..))
+import           Test.QuickCheck.Random         (newQCGen)


 --------------------------------------------------------------------------------
@@ -78,7 +79,7 @@
 --------------------------------------------------------------------------------
 sample :: Arbitrary a => IO [a]
 sample = do
-    gen <- newStdGen
+    gen <- newQCGen
     return $ (unGen arbitrary) gen 512
